routes/index.js

```diff
@@ -1,5 +1,10 @@
 var express = require('express');
+var clientID = "692884782115-u4o2n8dco40hjqa18b1agl9492m05l1j.apps.googleusercontent.com";
+var GoogleAuth = require('google-auth-library');
+var auth = new GoogleAuth;
+var client = new auth.OAuth2(clientID, '', '');
 var router = express.Router();
+var currentEmail;
```

Comment:
If I'm reading this correctly, currentEmail is shared between all users of the site?! If one person signs out, currentEmail is set to "" for everyone? If one person logs in, currentEmail is set to that email for everyone